README.md

```diff
@@ -22,4 +22,4 @@ A .Net dynamic fake framework for creating all types of fake objects, mocks, stu
 * [Package](https://nuget.org/packages/FakeItEasy "FakeItEasy on NuGet")
 
 -
-FakeItEasy logo designed by Vanja Pakaski.
+FakeItEasy logo designed by [@vanpak](https://github.com/vanpak).

```

Comment:
Fantastic. I was on the verge of tweeting her this morning. Then I saw the activity on her Twitter account and reconsidered.